inc/class-wpseo-shortlinker.php

```diff
@@ -30,7 +17,23 @@ public function __construct( $version ) {
 	 * @return string The final URL.
 	 */
 	public function build_shortlink( $url ) {
-		return $url . '?utm_content=' . $this->version;
+		$php_version      = $this->get_php_version();
```

Comment:
Why not putting the values directly in the array instead of storing them into variables?